extensions/wikia/UserLogin/EmailConfirmationController.class.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+class EmailConfirmationController extends WikiaController {
+	/**
+	 * Attempt to confirm the user's email address and show success or failure
+	 */
+	public function postEmailConfirmation() {
```

Comment:
Correct, that's why I started the name with POST and I'm using POST on Mercury level. That's a great catch since calling GET here would actually make this work as well, thanks!